src/components/Events/Fall_GM/style.scss

```diff
@@ -1,62 +1,167 @@
+    p#cant-wait {
       text-align: center;
-      font-size: 1.75em;
+      margin: auto;
+      margin-top: 25px;
+      margin-bottom: 50px;
+      width: 90%;
     }
-  }
-}
 
-h3#fall-gm-zoom-release {
-  margin: auto;
-  margin-top: 25px;
-  width: 75%;
-  text-align: center;
-}
-p#cant-wait {
-  text-align: center;
-  margin: auto;
-  margin-top: 25px;
-  margin-bottom: 50px;
-  width: 100%;
-}
+    #fall-gm-2020-graphic {
+      display: block;
+      margin: auto;
+      margin-top: 100px;
+      width: 85vw;
+    }
 
-#fall-gm-2020-graphic {
-  display: block;
-  margin: auto;
-  margin-top: 100px;
-  width: 85vw;
+    @media (max-width: 915px) {
+      .countdown-cards { height: 175px; }
+      #countdown-wrapper { margin-bottom: 125px;}
+    }
+  }
+
+  #gm-now-wrapper {
+    #time-to-gm-headline-now {
+      font-size: 3em;
+      text-align: center;
+    }
+    #gm-main-content{
+      margin: auto;
+      width: 75%;
+      #join-links-wrapper {
+        display: flex;
+        justify-content: space-around;
+        padding-bottom: 50px;
+        margin-bottom: 25px;
+        border-bottom: 1px dotted $acm-black;
+        .click-to-join{
+          .join-links{
+            border-radius: 10px;
+            .join-links-img{
+              display: block;
+              margin: auto;
+              height: auto;
+            }
+          }
+          #zoom-join-link-wrapper {
+            background-color: #2D8CFF;
+            padding-top: 20px;
+            padding-bottom: 20px;
+            .join-links-img {
+              width: 150px;
+            }
+          }
+          #discord-join-link-wrapper {
+            background-color: #7289DA;
+            padding-top: 5px;
+            padding-bottom: 1px;
+            .join-links-img {
+              width: 200px;
+            }
+          }
+        }
+        .click-to-join:hover{
+          opacity: 0.75;
+          cursor: pointer;
+        }
+        p.join-us {
+          text-align: center;
+        }
+      }
+      #gm-program-wrapper {
+        .row {
+          display: flex;
+          margin-top: 25px;
+          margin-bottom: 25px;
+          .timestamps{
+            margin: 0px;
+            padding-top: 3px;
+            min-width: 105px;
+            white-space: nowrap;
+          }
+          .row-description-wrapper{
+            margin-left: 350px;
+            h3.row-description-heading {
+              margin: 0px;
+              color: $acm-cobalt;
+            }
+            p.row-description-subheading {
+              margin-top: 4px;
+              color: $acm-tint;
+            }
+            #studio { color: $studio-raspberry; }
+            #icpc { color: $icpc-tangerine; }
+            #design { color: $design-orange; }
+            #cyber{ color: $cyber-amber; }
+            #teach { color: $teachla-green; }
+            #w { color: $w-teal; }
+            #ai { color: $ai-arctic; }
+            #hack { color: $hack-purple; }
```

Comment:
These are variables declared in `App.scss`.

We pull the trigger on rebranding on Sunday, October 4th. Assuming all we need to change is the color palette in `App.scss` and the committee assets. The color scheme on the program page will also update to the new branding